src/sidebar/components/version-info.js

```diff
@@ -6,8 +6,16 @@ import { withServices } from '../util/service-context';
 
 import Button from './button';
 
+/**
+ * @typedef VersionInfoProps
+ * @prop {import('../util/version-data')} versionData - Object with version information
```

Comment:
```suggestion
 * @prop {import('../util/version-data').default} versionData - Object with version information
```

I'm not certain about the reason, but it appears that in this context a default export needs to be referred to explicitly rather than using something like the syntax for an `import` statement.

Reply:
awe, gotcha